Package.swift

```diff
@@ -2,6 +2,24 @@
 
 import PackageDescription
 
+var targets: [Target] = []
+#if !os(macOS)
+targets += [
+    .systemLibrary(name: "zlib", path: "sources/zlib", pkgConfig: "zlib"),
+    .target(name: "PNG", dependencies: ["zlib"], path: "sources/png"),
+]
+#else
+targets += [
+    // Without “zlib”.
+    .target(name: "PNG", path: "sources/png"),
+]
+#endif
+targets += [
+        .target(name: "PNGTests", dependencies: ["PNG"], path: "tests"),
+        .target(name: "PNGBenchmarks", dependencies: ["PNG"], path: "benchmarks"),
+        .testTarget(name: "PNGXCTests", dependencies: ["SDGExternalProcess"], path: "XCTest/PNGTests")
```

Comment:
Let’s just do the simple thing, get rid of this, and run the test executable in the travis script,, otherwise I think this can be merged